modules/rest_api/controllers/services/rest.php

```diff
@@ -787,6 +787,61 @@ private function checkAllowedResource($proj_id, $resourceName) {
     ['caption' => 'Media', 'field' => '#occurrence_media#'],
   ];
 
+  /**
```

Comment:
I wonder if it would be better to have a single $esCsvTemplates associative array, keyed by the template name ('default' and 'easy-download' for now). This would remove the need for switch statements with a hard-coded list of templates.

Reply:
Agreed - I will implement.